packages/evolution-frontend/src/components/inputs/__tests__/InputChoiceSorting.test.ts

```diff
@@ -4,7 +4,9 @@
  * This file is licensed under the MIT License.
  * License text available at https://opensource.org/licenses/MIT
  */
+import exp from 'constants';
```

Comment:
These imports look like automatic vscode auto-completed imports, I'm don't think they're used, same for 'ro' below